### PR TITLE
[Core] Adding new test for `ParallelFillCommunicator`

### PR DESCRIPTION
**Pull request by loumalouomega**
**📝 Description**

Adding new test while I debug this.

**🆕 Changelog**

- [New Triforce test](https://github.com/KratosMultiphysics/Kratos/commit/a3cad88ab93d727665c1f0cb486c8f725e2c9969)

### CODE REVIEW

File: kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h [hunk 1]
```diff
@@ -47,6 +47,18 @@ static void GenerateDistributedTriangleMesh(
...
     );
 
+/**
+* @brief It generates a Triforce mesh
+* @details  ▲
+*          ▲ ▲
+* @param rModelPart The model part to be filled
+* @param rDataCommunicator The data communicator
+*/
+static void GenerateDistributedTriforceMesh(
```
Comment (rubenzorrilla) [inline kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: 🤦‍♂️ la madre que te parió...

**Review by rubenzorrilla — APPROVED** [review]
Further testing. Looks good to me.

File: kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h (hunk 1 above)
Comment (loumalouomega) [reply kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: Casualmente tenia un problema que fallaba con una distribucion similar. Lo podria haber llamado de otra manera, como ClanHojo test: https://es.wikipedia.org/wiki/Clan_H%C5%8Dj%C5%8D, es el origen de la trifuerza de acuerdo con Miyamoto san.

actually-nerd

Comment (philbucher) [reply kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: https://github.com/KratosMultiphysics/Kratos/pull/12726#discussion_r1787674758

:stuck_out_tongue:

Comment (loumalouomega) [reply kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: I didn't start in Spanish...

Comment (loumalouomega) [reply kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: Or we can switch to German/Spanish and come back to Charles V unity

<details>
Charles V
</details>

Comment (philbucher) [reply kratos/mpi/tests/test_utilities/mpi_cpp_test_utilities.h:57]: > I didn't start in Spanish...

I didnt mean you :D

**Pull request merged**